articles/node.js-vs-python/index.md

```diff
@@ -0,0 +1,123 @@
+
+To make a good decision on which technology to use, we will  consider the following criteria:
+1. Speed
+2. Scalability
+3. Universality
+4. Extensibility
+5. Architecture
+
+### SPEED
+Performance and Speed are the main features that every developer wishes his project to have at the end. Every developer aims to deliver a high-performance application to the user.
+
+#### Node.Js
+Since Node.js uses [V8 Engine](https://en.wikipedia.org/wiki/V8_(JavaScript_engine)) to interpret JavaScript code, its performance is remarkable. V8 is a powerful JavaScript engine that is provided by Google. It converts JavaScript code into machine code that will be understood by the processor.
+
+Node.js is also very responsive and has a very high loading speed.
+
+#### Python
+Unlike Java which is a compiled language, Python is an interpreted language. It is generally slower than the compiled languages. This makes Python lose to Node.js in terms of performance.
+
+Python is not suitable for apps that require high execution speed. This is because of the single flow of code in Python which leads to slow processing of requests. Python web applications are therefore slower.
+
+### SCALABILITY
+Scalability is the ability of a web app to adapt to changes like an increase in performance, new features, etc.
+
+#### Node.Js
+Unlike Python, Node.js uses different modules for its functionality. Therefore, for the increase of scalability of an app developed by Node.js few modules are added. This will increase the demand for using Node.js by developers.
+
+Node.js is an asynchronous technology. (can run multiple processes)Hence it facilitates multithreading.
+#### Python
+Python does not support multithreading. Therefore, scalability is not easy. For Python to have easy scalability, libraries have to be used. However, this does not mean that it can compete with Node.js in terms of scalability.
+
+### UNIVERSALITY
+
+ #### Node.Js
+Node.js is used in backend development. For frontend development, developers prefer JavaScript. This saves on resources since only one team with Node.js skills is hired. This feature of Node.js has, in turn, led many developers to choose it.
+
+Node.js is cross-stage. This means that a developer can create one desktop app that will work On different operating systems like MAC OS, Linux, and Windows.  This will be a great advantage to the developer in terms of the project cost.
+
+#### Python
+Python is a full-stack language. It is used in backend development while its frameworks are used in frontend development. 
+
+A python program can be written in MAC OS and the same program run in Linux, therefore Python is also a cross-stage
+
+Python is a good language for web development as well as desktop development, but unlike Node.js it is never used in mobile app development. 
```

Comment:
I can argue out that, although to a much lesser extent, Python can be, and is used for mobile app development. Please restructure this statement to remove the word 'never'.